### DISCUSSION TITLE
For images, a way to set the caption and credit for photos?

### DISCUSSION THREAD

**Discussion by kookster** [Podcastindex-org/podcast-namespace#455] [Ideas]
I know there is thinking about credits for the show in general, but for the channel and item images, how about a way to provide a caption and credit for the image/photos?

Especially for items, the episode image for news-y podcasts is often a photo which it would be great to have these both to respect the source of the image, and for accessibility this caption can function as an alt tag for the images.

**Comment by theDanielJLewis** [comment]
I know you're referring to episode images, but this kind of thing could be supported for chapter images if #400 would be accepted.

**Comment by daveajones** [comment]
An attribute in the tag that links over to a `<podcast:person>` tag perhaps?

**Comment by daveajones** [comment]
This also makes me think it might be helpful to have some "universal" attributes for linkages between all tags.  Perhaps an `id` attribute with a corresponding `ref` attribute for something like this:

```xml
<podcast:person id="7359051a-c75d-11ed-afa1-0242ac120002" ...>

<podcast:value ...>
    <podcast:valueRecipient ref="7359051a-c75d-11ed-afa1-0242ac120002" ...>
</podcast:value>
```

...where something like that indicates that "this" value recipient is for "that" person.
